tests/posix/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_PTHREAD_IPC=y
```

Comment:
that collection of tests should be in a subdirectory as well,, we should not have it at this level, call it tests/posix/common or something like that.

Reply:
are you planning to move everything else into this one single test?